src/yb/yql/pgwrapper/pg_libpq-test.cc

```diff
@@ -190,6 +190,36 @@ TEST_F(PgLibPqTest, Simple) {
   ASSERT_EQ(row, (decltype(row){1, "hello"}));
 }
 
+TEST_F(PgLibPqTest, PgStatIdxScanNoIncrementOnErrorTest) {
+  auto conn = ASSERT_RESULT(Connect());
+
+  std::string createTableStmt = "create table many (";
+  for (int i = 1; i <= 30; ++i)
+    createTableStmt += Format("c$0 int,", i);
+  createTableStmt += "k int primary key)";
+
+  std::string createIndexStmt = "create index on many (";
+  for (int i = 1; i <= 30; ++i)
+    createIndexStmt += Format("c$0,", i);
+  createIndexStmt += "c1 asc)";
```

Comment:
c1 shows up twice.  Logic can avoid that from happening, for example tracking whether it's the first iteration or not to determine whether comma should be added.  The first column in the index should have explicit "asc" in the syntax; the rest are implicit asc so that is not necessary, but you can still do that if lazy.